dist/keycode.cjs.js

```diff
@@ -2,123 +2,123 @@
 
 Object.defineProperty(exports, '__esModule', { value: true });
 
-const KEY_CANCEL = 3;
-const KEY_HELP = 6;
-const KEY_BACK_SPACE = 8;
-const KEY_TAB = 9;
-const KEY_CLEAR = 12;
-const KEY_RETURN = 13;
-const KEY_ENTER = 14;
-const KEY_SHIFT = 16;
-const KEY_CONTROL = 17;
-const KEY_ALT = 18;
-const KEY_PAUSE = 19;
-const KEY_CAPS_LOCK = 20;
-const KEY_ESCAPE = 27;
-const KEY_SPACE = 32;
-const KEY_PAGE_UP = 33;
-const KEY_PAGE_DOWN = 34;
-const KEY_END = 35;
-const KEY_HOME = 36;
-const KEY_LEFT = 37;
-const KEY_UP = 38;
-const KEY_RIGHT = 39;
-const KEY_DOWN = 40;
-const KEY_PRINTSCREEN = 44;
-const KEY_INSERT = 45;
-const KEY_DELETE = 46;
-const KEY_0 = 48;
-const KEY_1 = 49;
-const KEY_2 = 50;
-const KEY_3 = 51;
-const KEY_4 = 52;
-const KEY_5 = 53;
-const KEY_6 = 54;
-const KEY_7 = 55;
-const KEY_8 = 56;
-const KEY_9 = 57;
-const KEY_SEMICOLON = 59;
-const KEY_EQUALS = 61;
-const KEY_A = 65;
-const KEY_B = 66;
-const KEY_C = 67;
-const KEY_D = 68;
-const KEY_E = 69;
-const KEY_F = 70;
-const KEY_G = 71;
-const KEY_H = 72;
-const KEY_I = 73;
-const KEY_J = 74;
-const KEY_K = 75;
-const KEY_L = 76;
-const KEY_M = 77;
-const KEY_N = 78;
-const KEY_O = 79;
-const KEY_P = 80;
-const KEY_Q = 81;
-const KEY_R = 82;
-const KEY_S = 83;
-const KEY_T = 84;
-const KEY_U = 85;
-const KEY_V = 86;
-const KEY_W = 87;
-const KEY_X = 88;
-const KEY_Y = 89;
-const KEY_Z = 90;
-const KEY_LEFT_CMD = 91;
-const KEY_RIGHT_CMD = 93;
-const KEY_CONTEXT_MENU = 93;
-const KEY_NUMPAD0 = 96;
-const KEY_NUMPAD1 = 97;
-const KEY_NUMPAD2 = 98;
-const KEY_NUMPAD3 = 99;
-const KEY_NUMPAD4 = 100;
-const KEY_NUMPAD5 = 101;
-const KEY_NUMPAD6 = 102;
-const KEY_NUMPAD7 = 103;
-const KEY_NUMPAD8 = 104;
-const KEY_NUMPAD9 = 105;
-const KEY_MULTIPLY = 106;
-const KEY_ADD = 107;
-const KEY_SEPARATOR = 108;
-const KEY_SUBTRACT = 109;
-const KEY_DECIMAL = 110;
-const KEY_DIVIDE = 111;
-const KEY_F1 = 112;
-const KEY_F2 = 113;
-const KEY_F3 = 114;
-const KEY_F4 = 115;
-const KEY_F5 = 116;
-const KEY_F6 = 117;
-const KEY_F7 = 118;
-const KEY_F8 = 119;
-const KEY_F9 = 120;
-const KEY_F10 = 121;
-const KEY_F11 = 122;
-const KEY_F12 = 123;
-const KEY_F13 = 124;
-const KEY_F14 = 125;
-const KEY_F15 = 126;
-const KEY_F16 = 127;
-const KEY_F17 = 128;
-const KEY_F18 = 129;
-const KEY_F19 = 130;
-const KEY_F20 = 131;
-const KEY_F21 = 132;
-const KEY_F22 = 133;
-const KEY_F23 = 134;
-const KEY_F24 = 135;
-const KEY_NUM_LOCK = 144;
-const KEY_SCROLL_LOCK = 145;
-const KEY_COMMA = 188;
-const KEY_PERIOD = 190;
-const KEY_SLASH = 191;
-const KEY_BACK_QUOTE = 192;
-const KEY_OPEN_BRACKET = 219;
-const KEY_BACK_SLASH = 220;
-const KEY_CLOSE_BRACKET = 221;
-const KEY_QUOTE = 222;
-const KEY_META = 224;
+var KEY_CANCEL = 3;

```

Comment:
var ?

Reply:
Is this not generated?

Reply:
I used the `typescript` plugin now and this is that it generated, previously it was building to es6 that's why it was using const. Now the target is es5 as per `tsconfig.json`.

Reply:
Hmm.. `target: es5` was already there before.

Reply:
Before, it wasn't using typescript to build actually; rollup by default (w/o plugins) doesn't use typescript. 

Anway, now this is resolved.